experimental/apis/data/v0alpha1/query.go

```diff
@@ -37,12 +37,12 @@ type DataQuery struct {
 	CommonQueryProperties `json:",inline"`
 
 	// Additional Properties (that live at the root)
-	additional map[string]any `json:"-"` // note this uses custom JSON marshalling
+	Additional map[string]any `json:"-"` // note this uses custom JSON marshalling
```

Comment:
making this public would mean that you could set fields directly... even if common ☹️ 

Is the goal to fix deep copy issues?  if yes, lets see where else we could do that